static/manual_body.Rmd

```diff
@@ -1,237 +0,0 @@
----
```

Comment:
It's alright. Robin might not like the idea, but I believe we don't know the markdown document any more. HTML will serve the purpose fairly well.